README.md

```diff
@@ -200,3 +200,12 @@ The teacherservices.cloud domain is created in route53 and owned by infra-ops. S
 - [Low priority app](documentation/lowpriority-app.md)
 - [Monitoring](documentation/monitoring.md)
 - [Slack webhook integration](documentation/slack-webhook-integration.md)
+
+### PROMETHEUS , ALERTMANAGER and THANOS Auth Key generation.
```

Comment:
Would it be better in https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/monitoring.md ?

Reply:
moved this part to monitoring.md